scripts/release_scripts/update_changelog_and_credits.py

```diff
@@ -447,6 +453,70 @@ def create_branch(
             target_branch, current_release_version_number))
 
 
+def is_invalid_email_present(release_summary_lines):
+    """Checks if any invalid email is present in the list of
+    new authors and contributors.
+
+    Args:
+        release_summary_lines: list(str). List of lines in
+            ../release_summary.md.
+
+    Returns:
+        bool. Whether invalid email is present or not.
+    """
+    authors_start_index = release_summary_lines.index(
+        release_constants.NEW_AUTHORS_HEADER) + 1
+    authors_end_index = release_summary_lines.index(
+        release_constants.EXISTING_AUTHORS_HEADER) - 1
+    new_authors = release_summary_lines[authors_start_index:authors_end_index]
+    contributors_start_index = release_summary_lines.index(
+        release_constants.NEW_CONTRIBUTORS_HEADER) + 1
+    contributors_end_index = release_summary_lines.index(
+        release_constants.EMAIL_HEADER) - 1
+    new_contributors = (
+        release_summary_lines[contributors_start_index:contributors_end_index])
+    new_email_ids = new_authors + new_contributors
+    return any(
+        release_constants.INVALID_EMAIL_SUFFIX in id
```

Comment:
Maybe instead of just returning this, you can also print which ids are incorrect so it is easy to find and fix it.

Reply:
Done.